src/cinema/domainmodel/Seat.java

```diff
@@ -0,0 +1,41 @@
+package cinema.domainmodel;
+
+import java.util.UUID;
+
+public class Seat {
+    public Seat(int row, int column) {
+        this.row = row;
+        this.column = column;
+        this.price = row < 5 ? 10 : 8;
```

Comment:
I would recommend moving that logic into generate function to make this class simple POJO.